src/app/views/company/team-list/team-list.directive.coffee

```diff
@@ -102,11 +102,14 @@ DashboardOrganizationTeamListCtrl = ($scope, $window, $uibModal, $q, MnoeCurrent
       )
   )
 
+  #Setting third argument of $watch to true for deep watching
```

Comment:
[Use one space between the leading `#` character of the comment and the text of the comment](https://github.com/bbatsov/ruby-style-guide#hash-space)